package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "group-income",
   "version": "0.0.1",
-  "contractsVersion": "0.1.6",
+  "contractsVersion": "0.1.10",
```

Comment:
Same as with: https://github.com/okTurtles/group-income/pull/1722#pullrequestreview-1622754878

Please set this to `0.2.0` and from now on don't bump any of the versions in `e2e-protocol`